manage.py

```diff
@@ -94,4 +95,7 @@ def prepare_kubernetes_db():
 
 
 if __name__ == "__main__":
+    host = app.config['SERVER_NAME'].split(":")[0]
+    port = app.config['SERVER_NAME'].split(":")[1]
```

Comment:
This will fail if the port is not specified in the configuration

Reply:
Also, the splitting is done twice instead of once

Reply:
@iamareebjamal It will take the default value i.e `127.0.0.1:5000` if we didn't specify the `SERVER_NAME` in `.env`

Reply:
```python
host_port = app.config['SERVER_NAME'].split(":")
host = host_port[0]
if len(host_port) > 1:
     port = host_port[1]
else:
    port = 80
```

Reply:
I don't think one should write a SERVER_NAME without mentioning the port in the .env file. But yes, it would be better to handle that case, maybe using a ternary operator.

Reply:
@iamareebjamal I think instead of hard-coding 80 here, it is better to store that also in a default config key.

Reply:
@SaptakS @iamareebjamal I am confused what I have to do finally :confused:

Reply:
@IamAdiSri You can use urlparse library to get host name and port number.
```
from urlparse import urlparse
server_name = app.config.get('SERVER_NAME')
host_name = urlparse(server_name).hostname
port = urlparse(server_name).port
```

Reply:
@ParthS007 Where you have modified the SERVER_NAME parameter in config.py, add a FALLBACK_PORT=80 parameter as well and use that here